src/d3a/d3a_core/sim_results/__init__.py

```diff
@@ -29,6 +33,23 @@ def is_buffer_node_type(area):
     return area['type'] == "InfiniteBusStrategy"
 
 
+def get_unified_area_type(area):
+    if is_pv_node_type(area):
+        return "PV"
+    elif is_load_node_type(area):
+        return "Load"
+    elif is_cell_tower_type(area):
+        return "CellTower"
```

Comment:
I know, this is here because of completeness, but we should think about deprecating this special type at some point (as it is only a loadhoursStreategy and the UI isnt using this anyway.

Reply:
Yes, made a ticket to be taken into account later: https://gridsingularity.atlassian.net/browse/D3ASIM-2993